src/librustc/session/config.rs

```diff
@@ -855,6 +866,7 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
                  "NAME=PATH"),
         opt::opt("", "sysroot", "Override the system root", "PATH"),
         opt::multi("Z", "", "Set internal debugging options", "FLAG"),
+        opt::opt_u("", "err-format", "How errors and other messages are produced", "human|json"),
```

Comment:
The abbreviation of err instead of just using error doesn't seem worth it to me; thoughts?